server.py

```diff
@@ -0,0 +1,46 @@
+import random
+import re
+
+from twisted.web.server import Site, NOT_DONE_YET
+from twisted.web.resource import Resource
+from twisted.internet import reactor, endpoints
+
+class Home(Resource):
+    isLeaf = True
+
+    def _delayedRender(self, request):
+
+        path = "/var/www/html/books.toscrape.com/"
+        filepath = '/'.join(request.postpath)
+        fname = path + filepath
+
+        with open(fname) as f:
+            s = f.read()
+            request.write(s)
+            request.finish()
+
+    def _responseFailed(self, err, call):
+        call.cancel()
+
+    def render_GET(self, request):
+
+        domain_name = re.search(
+            'domain(.*):8880',
```

Comment:
it's possible to get domain in a better way, with `urllib.parse.urlsplit` in python 3, or `from six.moves.urllib.parse import urlsplit` to be compatible with python 2 - it gets you a parsing of the URL, and this object has `netloc` property that you can use here. Or at least exclude port number from regexp

Reply:
With `netloc` I'll get `domain1:8880`, but obtain just 1, can I use the same regexp with `netloc`?

Reply:
it's fine to use the whole ``domain1:8880`` for random.seed